downloadTaskManagement.js

```diff
@@ -0,0 +1,271 @@
+import * as env from './env.js';
+import { querySudo as query, updateSudo as update } from '@lblod/mu-auth-sudo';
+import { uuid, sparqlEscapeString, sparqlEscapeDateTime, sparqlEscapeUri } from 'mu';
+
+export async function getTaskInfoFromRemoteDataObject(remoteDataObjectUri) {
+  const remoteDataObjectUriSparql = sparqlEscapeUri(remoteDataObjectUri);
+  //TODO this query is rather fragile, relying on the links between melding, job and task via non-documented properties, made by the download-url-service
+  const taskQuery = `
+    ${env.getPrefixes(['nie', 'prov', 'dct', 'task', 'adms', 'tasko', 'ext'])}
```

Comment:
That would be simpler. Just wanted to be a bit more precise with prefixes because we don't always use the same prefix or forget prefixes. Always sending everything seems overkill, but having to add them like this is also not ideal. Will change back to a single prefix inclusion for now.